packages/status-core/src/utils/public-key-to-color-hash.ts

```diff
@@ -0,0 +1,91 @@
+// TODO?: import only used; or just '@noble/secp256k1' instead
+import * as secp256k1 from 'ethereum-cryptography/secp256k1'
```

Comment:
Non-blocking:

1. Requiring just `'@noble/secp256k1` would reduce the size if we end up not depending on `ethereum-cryptography`2. 
2. Importing everything with `import * as secp256k1` could be a trap if the package ever ends up exporting more

<img width="639" alt="image" src="https://user-images.githubusercontent.com/13265126/169715761-a8b55093-d6ca-4c5c-bbd8-0747bd927012.png">